### DISCUSSION TITLE
„QSL by“ at the extended Logbook

### DISCUSSION THREAD

**Discussion by DO3OI** [magicbug/Cloudlog#3208] [Q&A]
Hey hello, 
I hope you are doing well.
I have a question.
At the beginning, I always selected the QSOs in the extended logbook and then clicked on ‘update from the callbook’ under actions. The update would follow, and the names of the operators would appear, along with the method of sending the QSL card if one was requested.
Currently, this is not working. The column fot „QSL by“ remains empty.
Does this have something to do with the update from QRZ.com?
Did i do something wrong?

**Comment by magicbug** [comment]
I recently swapped how callbook information was stored its now inside your account settings this could have broke this function will need to double check

Make sure you have defined your login information, and logout and back in again to flush session information

**Comment by DO3OI** [comment]
<img width="1680" alt="Bildschirmfoto 2024-10-16 um 16 40 25" src="https://github.com/user-attachments/assets/32aa7004-85fe-41af-a4ba-b4acbd2fc2f6">

The column for Name, State, Locator  is getting updated but not QSL by. That's why I thought that's not the problem.

**Comment by magicbug** [comment]
Oh, thanks that narrows it down, sorry for misunderstanding!

**Comment by DO3OI** [comment]
My English is not the best. So I've today sorry. 
Is it a mistake from me, about incorrect use that "qsl by" is empty?

**Comment by DO3OI** [comment]
It’s me again:
Is there any chance that this feature will return soon?
At the moment, it’s very tedious to check each individual profile on QRZ to see if, and how, a person prefers to receive QSL cards.
Honestly, it’s taking the fun out of it for me.
